### PR TITLE
metadata

### PR DESCRIPTION
**Pull request by tebeka**

### CODE REVIEW

File: carrow.go [hunk 1]
```diff
@@ -38,9 +38,11 @@ func NewField(name string, dtype DType) (*Field, error) {
 
 	field := &Field{ptr}
 
-	runtime.SetFinalizer(field, func(f *Field) {
-		C.field_free(f.ptr)
-	})
+	/*
```
Comment (yonidavidson) [inline carrow.go]: Why comment and not delete?

File: carrow.go [hunk 2]
```diff
@@ -60,9 +62,11 @@ func NewFieldList() (*FieldList, error) {
 
 	fieldList := &FieldList{ptr}
 
-	runtime.SetFinalizer(fieldList, func(f *FieldList) {
-		C.field_free(f.ptr)
-	})
+	/*
```
Comment (yonidavidson) [inline carrow.go]: Why comment and no delete?

File: carrow.go (hunk 1 above)
Comment (tebeka) [reply carrow.go]: Agree, fixed.

File: carrow.go (hunk 2 above)
Comment (tebeka) [reply carrow.go]: Fixed.

**Pull request merged**